### ISSUE TITLE
怎么在shadowsocks调试？

### ISSUE THREAD

**Issue by gt11799** [gfwlist/tsilwfg#6]
我用的浏览器插件Switchy Omega上是可以用的，但是在shadowsocks却不生效，请问一下，我该怎么调试？

**Comment by cicku** [comment]
Are you inside the GFW or not?

**Comment by jacky441** [comment]
how to use this ?
I have ssr service setup on my router and I have ssr server vps in China to unblock youku etc....

**Comment by gt11799** [comment]
> Are you inside the GFW or not?

yeah, I'm inside the GFW, but I want to help my friend in the US to visit the site of China
